src/main/java/uk/gov/hmcts/reform/feature/security/UserDetailsConfigurer.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.feature.security;
+
+import org.springframework.security.config.annotation.authentication.configurers.provisioning.UserDetailsManagerConfigurer;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.UserDetailsManager;
+import rx.functions.Action3;
+import uk.gov.hmcts.reform.feature.webconsole.Ff4jUsersConfig;
+import uk.gov.hmcts.reform.feature.webconsole.Ff4jUsersConfig.UserDetails;
+
+import java.util.List;
+
+import static uk.gov.hmcts.reform.feature.security.Roles.ADMIN;
+import static uk.gov.hmcts.reform.feature.security.Roles.EDITOR;
+import static uk.gov.hmcts.reform.feature.security.Roles.USER;
+
+public class UserDetailsConfigurer {
+
+    private final UserDetailsManager manager;
+
+    private final Action3<String, String, String[]> createUser;
+
+    public UserDetailsConfigurer(UserDetailsManagerConfigurer<?, ?> configurer, PasswordEncoder passwordEncoder) {
+        this.manager = configurer.getUserDetailsService();
+        this.createUser = (String username, String password, String... roles) -> configurer
+            .withUser(username)
+            .password(passwordEncoder.encode(password))
+            .roles(roles);
```

Comment:
this function is always the same, called only **once**, and it's body is basically one line (formatted nicely).
We could just store `configurer` and `passwordEncoder` and use this functions body directly where it's used.